lib/generate_layout.js

```diff
@@ -11,14 +9,44 @@ module.exports = pages => {
       <meta name="description" content="${page.description}">
       <meta name="viewport" content="width=device-width,initial-scale=1">
 	  <link href="/style.css" rel="stylesheet">
-	  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.15.0/themes/prism.css" integrity="sha256-/Kfdz9pXGPe+bFF+TtxHqbg6F9c3Rb0jN48uy+2b/do=" crossorigin="anonymous" />
 	</head>
 	<body>
 	  <nav>${navigation}</nav>
 	  <main>
-		  <h1>${page.heading}</h1>
+		  <h1>${page.title}</h1>
 		  ${content}
 	  </main>
 	</body>
   </html>`;
 };
+
+// TODO: Implement a real generateNavigation function
```

Comment:
I'm not sure how to iterate over the tree to get a navigation like this, @FND . Haalp?

Reply:
```
▾ Tree
  ▾ children: Map<string, Page>  <-----+
    ▾ Page                             |
      ▸ children: Map<string, Page>  --+
```
So you could recursively iterate over `children` - which should probably be encapsulated in `Tree#walk` or `Tree#traverse`?

Or we might just generate while we're walking the tree within `generatePageTree` anyway (see `slug` concatenation; we might just add another option for recursion alongside `parent`).